gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HookHandler.java

```diff
@@ -592,6 +604,39 @@ public boolean handle(final RoutingContext ctx) {
         }
     }
 
+    /**
+     * Handles hook search requests based on the 'destination' property.
+     * Searches in both routes and listeners.
+     *
+     * @param queryParam the RoutingContext of the request
+     */
+    public void handleHookSearch(String queryParam,HttpServerResponse response) {
```

Comment:
I think `handleHookSearch` should be private. There is no need to provide this method outside of `HookHandler` class

Reply:
Ok.